### PR TITLE
Feature/draw semantic frame

### PR DESCRIPTION
**Pull request by PTavse**
The display colors for semantic keypoints and their bounding boxes are changed.

### CODE REVIEW

File: src/FrameDrawer.cc [hunk 1]
```diff
@@ -112,18 +112,18 @@ cv::Mat FrameDrawer::DrawFrame()
                 {
 					if(vCurrentKeys[i].class_id != -1)
 					{
-						if(vCurrentKeys[i].class_id ==255)
+						/*if(vCurrentKeys[i].class_id ==255)
```
Comment (paul-michalik) [inline src/FrameDrawer.cc]: Please remove inactive code

**Comment by PTavse** [comment]
@paul-michalik  Could you please have a look if the changes are good to go?

File: src/FrameDrawer.cc [hunk 2]
```diff
@@ -112,19 +112,9 @@ cv::Mat FrameDrawer::DrawFrame()
...
-						else
-						{
-							cv::rectangle(im,pt1,pt2,cv::Scalar(255,0,0));
-							cv::circle(im,vCurrentKeys[i].pt,2,cv::Scalar(255,0,0),-1);
-							gNewSemanticFeatureCount++;
-						}
-
+						cv::rectangle(im,pt1,pt2,cv::Scalar(0,0,255));
+						cv::circle(im,vCurrentKeys[i].pt,2,cv::Scalar(0,0,255),-1);
+						gNewSemanticFeatureCount++;
```
Comment (paul-michalik) [inline src/FrameDrawer.cc]: What is this variable for? Is it really global or is it just the name? Please get rid of global mutable variables, don't use them!

Comment (PTavse) [reply src/FrameDrawer.cc]: Yes Paul, I have removed the global variable in the code I am going to push under the issue https://github.com/paul-michalik/ORB_SLAM2/issues/31. (Show info about current frame)

**Comment by PTavse** [comment]
It draws the semantic keypoints and the bounding boxes in red color. Yes it is connected to issue #27

**Comment by PTavse** [comment]
@paul-michalik , could you please review? I have rebased the code and changed the semantic keypoints' color. Also I have removed a code which was no longer needed.

**Comment by paul-michalik** [comment]
Let's consider this done. Closing!

**Pull request closed without merging**